va/va.go

```diff
@@ -551,6 +559,13 @@ func (va *ValidationAuthorityImpl) logRemoteValidationDifferentials(
 	va.log.Infof("remoteVADifferentials JSON=%s", string(logJSON))
 }
 
+// remoteValidationResult is a struct that combines a problem details instance
+// (that may be nil) with the remote VA hostname that produced it.
+type remoteValidationResult struct {
+	Hostname string
```

Comment:
Let's call this "VAHostname" or "VA" to distinguish it from the hostname being validated.

Reply:
Thanks, resolved in 7807eec.